app/src/main/java/atlant/moviesapp/activity/TvShowDetails.java

```diff
@@ -163,7 +255,7 @@ public void onLongClick(View view, int position) {
     public void showCrew(List<Crew> crew) {
         String directorsString = "";
```

Comment:
Code for String manipulation should be put in a StringUtils class. Already commented on this, so please put all String manipulation code in a Helper class :)